nemo/collections/asr/audio_preprocessing.py

```diff
@@ -170,6 +153,7 @@ def __init__(
         window="hann",
         normalized=True,
     ):
+        self._sample_rate = sample_rate
```

Comment:
not really important - but any particular reason for making that variable an attribute of the instance?

Reply:
added